utils/telegraf/Makefile

```diff
@@ -59,12 +59,6 @@ define Package/telegraf-full/description
 	(Full build. including all plugins)
 endef
 
-define Package/telegraf/conffiles
-/etc/telegraf.conf
-endef
-
-Package/telegraf-full/conffiles = $(Package/telegraf/conffiles)
```

Comment:
Including /etc/telegraf.conf in `conffiles` means it gets preserved by opkg during package upgrades. You probably don't want to remove this section.

Reply:
Starting from 1.26.0 version `etc/telegraf.conf` is no longer included in repo:
https://github.com/influxdata/telegraf/pull/12766

It can be generated using `telegraf config` command though, ether with: `make config` or `telegraf config`

I fully agree that `telegraf.conf` has to be somehow preserved in package upgrades. I haven't figured out how to do it using OpenWrt Makefile.

Reply:
The config file can be preserved in package upgrades by keeping the `Package/telegraf/conffiles` and `Package/telegraf-full/conffiles` definitions in the makefile.